mslib/mscolab/server.py

```diff
@@ -351,6 +356,40 @@ def get_user():
     return json.dumps({'user': {'id': g.user.id, 'username': g.user.username}})
 
 
+@APP.route('/upload_profile_image', methods=["POST"])
+@verify_user
+def upload_profile_image():
+    user_id = request.form['user_id']
+    file = request.files['image']
+    if file and allowed_file(file.filename):
+        if file.mimetype.startswith('image/'):
+            try:
+                data = file.read()
+                if len(data) > mscolab_settings.MAX_IMAGE_SIZE:
+                    return jsonify({'message': 'File too large'}), 413
+                success, message = fm.save_user_profile_image(user_id, data)
```

Comment:
instead of adding to the database you can use existing methods to store it as file on the server and save its name to the db, https://github.com/Open-MSS/MSS/blob/develop/mslib/mscolab/chat_manager.py#L100
e.g. move the function to the filemanager and refactor accordingly

Reply:
The current code can be used to add malicious content into the database.